sections/footer.liquid

```diff
@@ -60,6 +60,9 @@
       <div class="footer__copyright footer__copyright--align-{{ section.settings.align_copyright }}">
         {% if section.settings.show_copyright %}
           &copy; {{ "now" | date: "%Y" }}, {{ shop.name }}
+          {% if settings.cn_activated %}
```

Comment:
I think that `cn_activated` could be named a bit more clear, it is hard to guess what "cn" stands for (cartoon network?), but I guess that ship has already sailed.

Reply:
`cn` is also a international country code for China (naming is hard) :)

Reply:
LOL, it stands for Cookie Notice, but I agree that the naming is weird.

I'm down to change it before this goes live. I think it's going to be easier to do now than later. We need a good name though. Let's try finding a good one in the standup maybe?